LDM-148.tex

```diff
@@ -396,6 +400,24 @@ \section{Prompt Base Enclave}\label{prompt-base-enclave}
 The current design exposes each service as a separate entity with potentially different service levels.
 A DMCS component still exists, but it is an implementation feature of the services' design rather than an architectural component.
 
+The overall service level for this enclave should be high, with the goal being to have the enclave functional whenever the Observatory is taking data.
+
+Of the services within the enclave, the Header Generator component of the Archiving service is critical to observing and should have a target uptime equal to the Engineering and Facility Database.
+
+The Observatory Operations Data, OCS Driven Batch, Prompt Processing Ingest, and Telemetry Gateway services are important for maintaining high-quality observing; lack of these services constitutes a degraded mode of operation without useful feedback loops.
+The stability of the telescope systems is expected to require Observatory Operations Data and OCS Driven Batch to be down for no more than 24 hours at a time.
+Similarly, DMS-REQ-0008 requires that the pipelines fed by Prompt Processing Ingest and producing output via the Telemetry Gateway be down for no more than 24 hours at a time.
+After such an outage, the service must be available for at least 24 hours before going down again.
+In both cases, there are exceptions for observatory scheduled maintenance, although it should be noted that part of the restoration of observing after the annual maintenance period requires that DM services be available, so only part of that maintenance period can be used for DM service maintenance.
+Also note that operations plans call for a scheduled DM maintenance period each morning.
+The overall Archiving service can prevent further observing if the capacity of the buffer in the Camera Data System (DAQ) at the Summit is exceeded, so it must be up sufficiently to drain that buffer.
```

Comment:
Doesn't the Archiving service also feed OODS so it has OODS's availability requirements (along with the previously mentioned Header Generator being part of this service)?    Maybe this sentence really belongs in the Data Backbone Section?

Reply:
I'm a little reluctant to constrain Archiving by the OODS, as it was originally conceived as a low-reliability, best-effort alternative to Prompt Processing, but I acknowledge that we have perhaps changed this by using it as the source for the OODS and CC.